### ISSUE TITLE
V4 Docs - need clearfix on content/typography horiz description example?

### ISSUE THREAD

**Issue by tshinnic** [twbs/bootstrap#17969]
While reading the v4-alpha1 docs closely on mobile, I noticed the "Horizontal description" dl/dt/dd example was sometimes misplacing dt texts vertically at xs/sm tiers.  After playing with this it looks like the simple use of "col-sm-3" gets confused when the text of a prior "col-sm-3" wraps within its area.

 &nbsp; &nbsp; &nbsp; ![content_typo_horiz_euismod_nok_5](https://cloud.githubusercontent.com/assets/842881/10568205/6756d864-75da-11e5-9e91-f54a2db048da.png)

When I use what I think is meant by the 'clearfix' then the problem is avoided.

 &nbsp; &nbsp; &nbsp; ![content_typo_horiz_euismod_ok_4](https://cloud.githubusercontent.com/assets/842881/10568208/7a1135a8-75da-11e5-9645-c17319be2cc6.png)

```
--- a/docs/content/typography.md
+++ b/docs/content/typography.md
@@ -224,6 +224,8 @@ Align terms and descriptions horizontally by using our grid system's predefined
   <dt class="col-sm-3">Description lists</dt>
   <dd class="col-sm-9">A description list is perfect for defining terms.</dd>

+  <div class="clearfix"></div>
+
   <dt class="col-sm-3">Euismod</dt>
   <dd class="col-sm-9">Vestibulum id ligula porta felis euismod semper eget lacinia odio sem nec elit.</dd>
   <dd class="col-sm-9 col-sm-offset-3">Donec id elit non mi porta gravida at eget metus.</dd>
@@ -231,6 +233,8 @@ Align terms and descriptions horizontally by using our grid system's predefined
   <dt class="col-sm-3">Malesuada porta</dt>
   <dd class="col-sm-9">Etiam porta sem malesuada magna mollis euismod.</dd>

+  <div class="clearfix"></div>
+
   <dt class="col-sm-3 text-truncate">Truncated term is truncated</dt>
   <dd class="col-sm-9">Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus.</dd>
 </dl>
```

Is this change "worth it" to avoid the unsightliness?  Would this in fact be a good example of when to use the 'clearfix' thing?

**Comment by bassjobsen** [comment]
#18669 should close this?

**Comment by cvrebert** [comment]
Even after having merged #18669, there is still a similar bug in the example.

**Comment by patrickhlauke** [comment]
ref https://github.com/twbs/bootstrap/issues/18652#issuecomment-167236962 for possible clearfix solution (not tested it myself yet)

**Comment by mdo** [comment]
Fix coming in #19104.
